### PR TITLE
[WIP] quick and dirty table of connections by platform

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
Related to #40011 
This is a proposed short-term solution. Please review for accuracy and also for 

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

##### ANSIBLE VERSION
2.5

### CODE REVIEW

**Review by gundalow — COMMENTED** [review]
Please add asa
Not supported 
Network_cli since 2.5
No netconf 
No http
Local since 2.4

**Comment by gundalow** [comment]
Rather than `no` I wonder if we should use `n/a`, to make it clearer that its not missing Ansible functionally, it that the option doesn't make sense for that particular platform

File: docs/docsite/rst/network/user_guide/platform_index.rst [hunk 1]
```diff
@@ -14,3 +14,23 @@ Some Ansible Network platforms support multiple connection types, privilege esca
...
+| IOS-XR*        | in v. >=2.5          | no                   | no               |  in v. >=2.4     |
++----------------+----------------------+----------------------+------------------+------------------+
+| JUNOS*         | in v. >=2.5          | in v. >=2.5          | no               |  in v. >=2.4     |
++----------------+----------------------+----------------------+------------------+------------------+
+| NXOS*          | in v. >=2.5          | no                   | in v. >=2.6      |  in v. >=2.4     |
++----------------+----------------------+----------------------+------------------+------------------+
+| VYOS*          | in v. >=2.5          | no                   | no               |  in v. >=2.4     |
++----------------+----------------------+----------------------+------------------+------------------+
+
+* Maintained by Ansible Network Team
```
Comment (gundalow) [inline docs/docsite/rst/network/user_guide/platform_index.rst]: Currently this is being rendered as a bullet point, 
```
`*` Maintained by Ansible Network Team
```

**Review by gundalow — COMMENTED** [review]
Please also add ASA

**Comment by gundalow** [comment]
This is a good step forward, thanks

File: docs/docsite/rst/network/user_guide/platform_index.rst [hunk 2]
```diff
@@ -14,3 +14,23 @@ Some Ansible Network platforms support multiple connection types, privilege esca
    platform_ios
    platform_junos
    platform_nxos
+
+Connections Settings by Platform 
+
++----------------+----------------------+----------------------+------------------+------------------+
+|..              | network_cli          | netconf              | httpapi          | local            |
```
Comment (gundalow) [inline docs/docsite/rst/network/user_guide/platform_index.rst]: Wonder if we could change the first column to `ansible_network_os`?

**Comment by acozine** [comment]
Superceded by #40688.

**Pull request closed without merging**